--- dconf-editor-3.22.0/editor/Makefile.am.appdata	2016-10-05 20:44:11.558721480 +0200
+++ dconf-editor-3.22.0/editor/Makefile.am	2016-10-05 20:44:24.062281504 +0200
@@ -68,6 +68,7 @@ ca.desrt.dconf-editor.service: Makefile
 	             echo 'Exec=${bindir}/dconf-editor --gapplication-service') > $@.tmp && \
 	             mv $@.tmp $@
 
+@APPSTREAM_XML_RULES@
 appstream_in_files = ca.desrt.dconf-editor.appdata.xml.in
 appstream_XML = $(appstream_in_files:.xml.in=.xml)
 @INTLTOOL_XML_RULE@
